pennylane/return_types.py

```diff
@@ -181,104 +181,8 @@ def circuit(a, b):
-          Array(-0.09784342, dtype=float32, weak_type=True)),
-         (Array(-0.09784339, dtype=float32, weak_type=True),
-          Array(-0.19767687, dtype=float32, weak_type=True)))
-
-        The new return types system also unlocks the use of ``probs`` mixed with different measurements with JAX:
-
-        .. code-block:: python
-
-            import jax
-
-            qml.enable_return()
-
-            dev = qml.device("default.qubit", wires=2)
-            qml.enable_return()
-
-            @qml.qnode(dev, interface="jax")
-            def circuit(a):
-              qml.RX(a[0], wires=0)
-              qml.CNOT(wires=(0, 1))
-              qml.RY(a[1], wires=1)
-              qml.RZ(a[2], wires=1)
-              return qml.expval(qml.PauliZ(wires=0)), qml.probs(wires=[0, 1])
-
-            x = jax.numpy.array([0.1, 0.2, 0.3])
-
-        >>> jax.jacobian(circuit)(x)
-        (Array([-9.9833414e-02, -7.4505806e-09,  6.9285655e-10], dtype=float32),
-         Array([[-4.9419206e-02, -9.9086545e-02,  3.4938008e-09],
-                      [-4.9750542e-04,  9.9086538e-02,  1.2768372e-10],
-                      [ 4.9750548e-04,  2.4812977e-04,  4.8371929e-13],
-                      [ 4.9419202e-02, -2.4812980e-04,  2.6696912e-11]],            dtype=float32))
-
-        where before the following error was raised:
-
-        .. code-block:: python
-
-            ValueError: All input arrays must have the same shape.
-
-        The new return types system also unlocks the use of shot vectors with all the previous features. For example you
-        can take the second derivative and multiple measurement with with JAX:
-
-        .. code-block:: python
-
-            import jax
-
-            qml.enable_return()
-
-            dev = qml.device("default.qubit", wires=2, shots=(1, 10000))
-
-            params = jax.numpy.array([0.1, 0.2])
-
-            @qml.qnode(dev, interface="jax", diff_method="parameter-shift", max_diff=2)
-            def circuit(x):
-                qml.RX(x[0], wires=[0])
-                qml.RY(x[1], wires=[1])
-                qml.CNOT(wires=[0, 1])
-                return qml.var(qml.PauliZ(0) @ qml.PauliX(1)), qml.probs(wires=[0])
-
-        >>> jax.hessian(circuit)(params)
-        ((Array([[ 0.,  0.],
-                        [ 2., -3.]], dtype=float32),
-          Array([[[-0.5,  0. ],
-                       [ 0. ,  0. ]],
-                      [[ 0.5,  0. ],
-                       [ 0. ,  0. ]]], dtype=float32)),
-         (Array([[ 0.07677898,  0.0563341 ],
-                       [ 0.07238522, -1.830669  ]], dtype=float32),
-          Array([[[-4.9707499e-01,  2.9999996e-04],
-                        [-6.2500127e-04,  1.2500001e-04]],
-                       [[ 4.9707499e-01, -2.9999996e-04],
-                        [ 6.2500127e-04, -1.2500001e-04]]], dtype=float32)))
-
     .. details::
-        :title: Usage Details for Autograd
+        :title: Autograd Usage Details
```

Comment:
Personally, I think having the titles be different in the first word can boost looking and browsing through the page as a whole.